pkg/acceptance/helpers/function_client.go

```diff
@@ -41,7 +41,7 @@ func (c *FunctionClient) CreateWithIdentifier(t *testing.T, id sdk.SchemaObjectI
 			id.SchemaObjectId(),
 			*sdk.NewFunctionReturnsRequest().WithResultDataType(*sdk.NewFunctionReturnsResultDataTypeRequest(sdk.DataTypeInt)),
 			"SELECT 1",
-		).WithArguments(argumentRequests),
+		).WithArguments(argumentRequests).WithSecure(true),
```

Comment:
I don't think that secure should be a default here

Reply:
This is required so that the functions can be granted privileges.

Reply:
Done.